app/assets/javascripts/mobile/profile_aspects.js

```diff
@@ -0,0 +1,74 @@
+$(document).ready(function(){
+ /* profile page: aspect-dropdown */
+  var person_id    = $('#user_aspects').data('person-id');
+
+  function profileRefreshAspectDropdown() {
+    var covertext, numselected = $('#user_aspects option.selected').length;
+    if(numselected === 0) {
+      $('#user_aspects').removeClass('hasconnection');
+      covertext = Diaspora.I18n.t('aspect_dropdown.add_to_aspect');
+    } else {
+      $('#user_aspects').addClass('hasconnection');
+      if(numselected === 1) {
+        covertext = $('#user_aspects option.selected').data('name');
+      } else {
+        covertext = Diaspora.I18n.t('aspect_dropdown.toggle', { 'count' : numselected });
+      }
+    }
+    $('#user_aspects option.listcover').text(covertext);
+  }
+
+  $('#user_aspects').change(function() {
+    var selected = $('#user_aspects option:selected');
+    $('#user_aspects option.listcover').text(Diaspora.I18n.t('aspect_dropdown.updating'));
+    $('#user_aspects').val('listcover'); // switch back to cover
+
+    if(selected.hasClass('selected')) {
+      // remove from aspect
+      var membership_id = selected.data('membership_id');
+      $.ajax({
+        url: '/aspect_memberships/' + membership_id,
```

Comment:
I know this is basically nowhere done yet, but let's start using [js-routes](https://github.com/railsware/js-routes) when we have it in the Gemfile already. You might need to add [the require](https://github.com/diaspora/diaspora/blob/develop/app/assets/javascripts/main.js#L6) for mobile, I didn't check.